tests/lean/redundant_pattern.lean.expected.out

```diff
@@ -1,4 +1,5 @@
 definition foo [forward] : ∀ (m n k : ℕ), P (f m) → P (g n) → P (f k) → P k ∧ R (g m) (f n) ∧ P (g m) ∧ P (f n) :=
 λ (m n k : ℕ), sorry
 (multi-)patterns:
-{P ?M_1, R (g ?M_2) (f ?M_3)}
```

Comment:
I see now. The pretty printer has a metavariable "purification" step where it renames metavariables. We can disable it using `set_option pp.purify_metavars false`